Dockerfile

```diff
@@ -83,8 +83,12 @@ COPY 10-htcondor.conf /etc/supervisord.d/
 COPY 50-main.config /etc/condor/config.d/
 RUN chmod 755 /bin/entrypoint.sh
 
-RUN [[ -n $TIMESTAMP ]] || TIMESTAMP=$(date +%Y%m%d-%H%M); \
-    sed -i "s|@CONTAINER_TAG@|opensciencegrid/osgvo-docker-pilot:${BASE_YUM_REPO}-${TIMESTAMP}|" \
+RUN if [[ -n $TIMESTAMP ]]; then \
+       tag=${BASE_YUM_REPO}-${TIMESTAMP} \
+     else \
+       tag=manual-build \
+    fi; \
+    sed -i "s|@CONTAINER_TAG@|opensciencegrid/osgvo-docker-pilot:$tag|" \
```

Comment:
I don't like unconditionally putting `opensciencegrid/osgvo-docker-pilot` in the tag. What if I make my own for testing and it's `matyasosg/osgvo-docker-pilot`?

Reply:
Good point. I'll just set the attr to the empty string if the TIMESTAMP build arg isn't specified